Add SIG Docs Spotlight blog article (#323)
kubernetes/contributor-site: content/en/blog/2022/sig docs spotlight blog.md

> # Spotlight on SIG-Docs

Reviewer: Hi - this needs front matter (see other files in this folder)

> The official documentation is the go-to source for any open source project. For Kubernetes, it's an ever-evolving Special Interest Group(SIG) with people constantly putting in their efforts to make details about the project easier to consume for new contributors and users . SIG Docs publishes the official documentation on kubernetes.io which includes, but is not limited to, documentation of the core APIs, core architectural details, and CLI tools shipped with the Kubernetes release.

Reviewer: ```suggestion
The official documentation is the go-to source for any open source project. For Kubernetes, it's an ever-evolving Special Interest Group(SIG) with people constantly putting in their efforts to make details about the project easier to consume for new contributors and users. SIG Docs publishes the official documentation on [kubernetes.io](https://kubernetes.io) which includes, but is not limited to, documentation of the core APIs, core architectural details, and CLI tools shipped with the Kubernetes release.
```

> To learn more about the work of SIG-Docs and its future ahead in shaping the community, I have summarised my conversation with the co-chairs, [Divya Mohan(DM)](https://twitter.com/Divya_Mohan02), [Rey Lejano(RL)](https://twitter.com/reylejano) and Natali Vlatko(NV), who walk us through the SIGs goals and how fellow contributors can help.

Reviewer: ```suggestion
To learn more about the work of SIG Docs and its future ahead in shaping the community, I have summarised my conversation with the co-chairs, [Divya Mohan](https://twitter.com/Divya_Mohan02) (DM), [Rey Lejano](https://twitter.com/reylejano) (RL) and Natali Vlatko (NV), who walk us through the SIG's goals and how fellow contributors can help.
``

**DM** : Another area where there has been a lot of success is the New Contributor Ambassador role, which has helped in making a more accessible point of contact for the onboarding of new contributors into the project.

**NV** : For each release cycle, SIG Docs have to review release docs and feature blogs highlighting release updates within a short window. This is always a big effort for the docs and blogs reviewers. 

### Q. Is there something exciting coming up for the future of SIG Docs that you want the community to know?

SIG Docs is now looking forward to establishing a roadmap, having a steady pipeline of folks being able to push improvements to the documentation and streamlining community involvement in triaging issues and reviewing PRs being filed. To build one such contributor and reviewership base, a mentorship program is being set up to help current contributors become reviewers. This definitely is a space to watch out for more!

### Wrap Up

> SIG Docs hosted a [talk on KubeCon + CloudNativeCon North America 2021](https://www.youtube.com/watch?v=GDfcBF5et3Q) with a deep dive into their awesome SIG. They are very welcoming and have been the starting ground into Kubernetes for a lot of new folks who want to contribute to the project. Join the [SIG's meetings](https://github.com/kubernetes/community/blob/master/sig-docs/README.md) to find out about the most recent research results, their plans for the forthcoming year, and how to get involved in the upstream Docs team as a contributor!

Reviewer: I'd write:
```suggestion
SIG Docs hosted a [deep dive talk](https://www.youtube.com/watch?v=GDfcBF5et3Q) during  on KubeCon + CloudNativeCon North America 2021, covering their awesome SIG. They are very welcoming and have been the starting ground into Kubernetes for a lot of new folks who want to contribute to the project. Join the [SIG's meetings](https://github.com/kubernetes/community/blob/master/sig-docs/README.md) to find out about the most recent research results, their plans for the forthcoming year, and how to get involved in the upstream Docs team as a contributor!
```
